ols/src/llms/llm_loader.py

```diff
@@ -331,14 +330,10 @@ def _watson_llm_instance(self) -> LLM:
 
         # WatsonX uses different parameter names
         llm_model = Model(
-            model_id=self.llm_params.get(
-                "model_id", os.environ.get("WATSON_MODEL", None)
-            ),
+            model_id=self.llm_params.get("model_id", None),
```

Comment:
`model_id` should be the same value as `model name` from our config object, not come from llm_params